app/messages/deployment_create_message.rb

```diff
@@ -7,6 +7,7 @@ class DeploymentCreateMessage < MetadataBaseMessage
       droplet
       revision
       strategy
+      options
```

Comment:
*thought/todo* I believe we need validation to ensure max_in_flight must be >= 1

Reply:
I noticed that we don't have validations that instances are >= 1...